weave/tests/test_basket.py

```diff
@@ -6,6 +6,7 @@
 import tempfile
 import shutil
 from pathlib import Path
+import sys
```

Comment:
Imports should be in alphabetical order. And all "From x import y" should be at the end of the import list.
So just move "import sys" to line 8 above "from pathlib import path"